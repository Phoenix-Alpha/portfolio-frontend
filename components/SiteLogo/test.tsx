import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { SiteLogo } from '.'

describe('<SiteLogo />', () => {
  it('should renders correct <SiteLogo />', () => {
    const wrapper = shallow(<SiteLogo />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
