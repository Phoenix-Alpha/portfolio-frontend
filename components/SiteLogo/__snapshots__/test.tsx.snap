// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SiteLogo /> should renders correct <SiteLogo /> 1`] = `
<Link
  href="/"
  passHref={true}
>
  <a
    className="flex items-center transition duration-1000 ease-in-out transform hover:scale-125"
  >
    <Image
      alt="logo"
      className="animate-pulse"
      height={42}
      src="/images/phoenix-logo.png"
      width={63}
    />
  </a>
</Link>
`;
